# Single trajectory point for control
float64 x
float64 y
float64 yaw
float64 velocity
float64 curvature
float64 time_from_start